recipes/pecat/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pecat" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+
+source:
+  git_url: https://github.com/lemene/PECAT.git
+  git_rev: 'v{{ version }}'
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - minimap2 >=2.17
+    - racon >=1.4.3
+    - perl >=5.26.2
+    - python >=3.6
+
+test:
+  commands:
+    - pecat.pl | grep 'unzip'
+
+about:
+  home: https://github.com/lemene/PECAT
+  license: BSD-2-Clause license
+  summary: A phased error correction and assembly tool
+
+extra:
+  skip-lints:
+    - missing_hash
+    - uses_vcs_url
```

Comment:
Is it possible to use conda versions of any of the submodule dependencies instead?

Reply:
It is recommended to not vendor other packages. Your thirdparty packages should be a dependency under the host section.

Reply:
OK, I see. Thank you for your suggestion. I have another question. Should I close the pull request, modify the code and then open a new pull request?

Reply:
You can change it in place here, no need for a new one.